cheeseshop/tests/fakes.py

```diff
@@ -44,5 +51,15 @@ def __init__(self, keystone_session, region_id, interface='public',
     async def get_object(self, name, container):
         return get_swift_object(name, container)
 
-    async def create_tempurl(self, name, container):
-        return 'http://some-swift.com/tempurl-unique'
+
+class FakeHmac(object):
+    def __init__(self, key, hmac_body, digest=sha1):
+        self.key = key
+        self.hmac_body = hmac_body
+        self.digest = digest
+        print ("IT ME {0}-{1}".format(self.key, self.hmac_body))
+
+    def hexdigest(self):
+        print ("IN YOUR DIGEST HEXING UR DUDES")
+        return "DAEDBEFFCAFE"
```

Comment:
DEADBEFF not DEADBEEF?

Reply:
This kind of code review is how we build quality into the product